packages/app-admin/src/styles/material.scss

```diff
@@ -38,5 +38,5 @@ body {
   --mdc-typography-font-family: #{$mdc-typography-font-family};
   --mdc-typography-subtitle2-font-weight: #{$mdc-typography-subtitle2-font-weight};
 
-  font-family: "Source Sans Pro";
+  //font-family: "Source Sans Pro";
```

Comment:
Cleanup or leave uncommented?

If you ask me, we can clean these up, and add reminders in GH project if required.

Reply:
👍